### PR TITLE
readme: development information

### PR DESCRIPTION
**Pull request by dhimmel**

### CODE REVIEW

**Comment by dhimmel** [comment]
@cthoyt did you want to take a quick look at the diff and see if this guidance makes sense?

File: README.md [hunk 1]
```diff
@@ -129,6 +129,35 @@ pip install nxontology[viz]
...
+
+# Set up the git pre-commit hooks.
+# `git commit` will now trigger automatic checks including linting.
+pre-commit install
+
+# Run all pre-commit checks (CI will also run this).
+pre-commit run --all
+
+# run tests
+pytest
```
Comment (cthoyt) [inline README.md:154]: I'd suggest using `tox` for the tests and virtual environment management, but this good either way

File: README.md [hunk 2]
```diff
@@ -129,6 +129,35 @@ pip install nxontology[viz]
...
+```
+
+Releases are created on [GitHub](https://github.com/related-sciences/target-ranking/releases).
```
Comment (cthoyt) [inline README.md:157]: fix this link

File: README.md (hunk 1 above)
Comment (dhimmel) [reply README.md:154]: Okay I'll play with tox and update the readme once I get the hang of it.

Comment (cthoyt) [reply README.md:154]: I can send an example as a PR

**Pull request merged**